arrows/core/video_input_image_list.cxx

```diff
@@ -128,13 +131,18 @@ ::get_configuration() const
 
   config->set_value( "path", "",
                      "Path to search for image file. "
-                     "If a file name is not absolute, this list of directories is scanned to find the file. "
-                     "The current directory '.' is automatically appended to the end of the path. "
-                     "The format of this path is the same as the standard "
-                     "path specification, a set of directories separated by a colon (':')" );
+                     "If a file name is not absolute, this list of directories "
+                     "is scanned to find the file. The current directory '.' is "
+                     "automatically appended to the end of the path. The format "
+                     "of this path is the same as the standard path specification, "
+                     "a set of directories separated by a colon (':')" );
   config->set_value( "allowed_extensions", "",
                      "Semicolon-separated list of allowed file extensions. "
                      "Leave empty to allow all file extensions." );
+  config->set_value( "sort_by_time", "false",
+                     "Instead of accepting the input list as-is, sort input file "
```

Comment:
Suggestion: "Rather than using the input list as-is, sort the list based on [the] timestamp contained in the file[s'] metadata." In particular, I would avoid suggesting that the file name is used somehow, since that is only the case if an additional layer which synthesizes file metadata based on the file name is in play (in which case it is "obvious" that the file name is the source of the metadata).

(Note: the "[]"s in the suggested text are optional suggestions, not intended to be literally included.)